src/main/java/io/cdap/plugin/servicenow/source/ServiceNowRecordReader.java

```diff
@@ -68,7 +72,7 @@ public void initialize(InputSplit split, TaskAttemptContext context) {
   public boolean nextKeyValue() throws IOException {
     try {
       if (results == null) {
-        fetchData();
+         fetchData();
```

Comment:
revert whitespace change

Reply:
Reverted. 7661e08

Reply:
Looks like this change didn't get reverted

Reply:
The whitespace change had been reverted.